modules/widgets/src/stylesheet.css

```diff
@@ -0,0 +1,33 @@
+.deckgl-wdgt {
+  margin: var(--wdgt-margin, 12px);
+}
+
+.deckgl-wdgt-button-border {
+  width: var(--button-size, 28px);
```

Comment:
`@deck.gl/widgets` won't define values for the CSS variables to avoid cascading issues (e.g. what happens when app css is loaded before lib css?). Instead, deck defines a default in each usage of `var()` according to the Light Flat theme.

Reply:
Yes... Just to clarify, when i said `deck.gl` I just meant to say that the deck core module should not be bothered with CSS registration type stuff (it should be handled the deck widget module and/or the app).

Reply:
Ah, ok yes. Totally agree. This approach leaves it up to the app to import the CSS stylesheet with whatever approach they use for other sheets. The widget module just provides an unattached style sheet and should come with usage docs similar to [maplibre](https://maplibre.org/maplibre-gl-js/docs/#maplibre-css).